recipes/libpfm4/all/conanfile.py

```diff
@@ -0,0 +1,60 @@
+from conan import ConanFile, tools
```

Comment:
need to import according to specialized tool, like

from conan.tools import files

or 

from conan.tools.files import get, rmdir, copy ...

Reply:
fixed